ctru-rs/src/services/gfx.rs

```diff
@@ -55,6 +55,28 @@ pub trait Screen: private::Sealed {
         unsafe { ctru_sys::gfxSetDoubleBuffering(self.as_raw(), enabled) }
     }
 
+    /// Swaps the video buffers.
```

Comment:
```suggestion
    /// Flushes the video buffer for this screen.
```

Reply:
Copying and pasting is always a bad habit... thanks for noticing (though I will look into all docs later on) 👍